src/shared/model/auth-slice/index.ts

```diff
@@ -0,0 +1,58 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import type { PayloadAction } from '@reduxjs/toolkit';
+
+export interface IAuthState {
+  isAutorized: boolean;
+  isLoading: boolean;
+}
+
+const initialState: IAuthState = {
+  isAutorized: false,
+  isLoading: false,
+};
+
+export const autorize = createAsyncThunk(
+  'user/autorize',
+  async (_, thunkAPI) => {
+    try {
+      await new Promise((resolve) => {
+        setTimeout(() => {
+          resolve(true);
+        }, 500);
+      });
+      return true;
+    } catch (error) {
+      return thunkAPI.rejectWithValue('autorize error');
+    }
+  }
+);
+
+export const authSlice = createSlice({
+  name: 'auth',
+  initialState,
+  reducers: {
+    logout: (state) => {
+      state.isAutorized = false;
+    },
+  },
+  extraReducers: (builder) => {
+    builder
+      .addCase(autorize.fulfilled, (state, action: PayloadAction<boolean>) => {
+        console.log('action', action.payload);
```

Comment:
Try this, should help you to log actions: https://github.com/LogRocket/redux-logger